packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -1133,9 +1150,11 @@ class Viewport implements IViewport {
    * Updates the actors clipping planes orientation from the camera properties
    * @param updatedCamera - ICamera
    */
-  protected updateClippingPlanesForActors(updatedCamera: ICamera): void {
+  protected async updateClippingPlanesForActors(
+    updatedCamera: ICamera
+  ): Promise<void> {
     const actorEntries = this.getActors();
-    actorEntries.forEach((actorEntry) => {
+    await actorEntries.forEach(async (actorEntry) => {
```

Comment:
what is async here that you need an await?

Reply:
Because if we remove the await, the 3d volume render will not correctly call the renderer.resetCameraClippingRange();

Reply:
But which part of the the inner functions is asynchronous? Do you know?

Reply:
Yes I understand what you are saying... I really dont know.... Any of these functionas appears to be the one. I remember if I removed this await if will not render correctly the volumeViewport3D. Maybe you can do this test again...